pkg/rbac/common.go

```diff
@@ -20,17 +20,12 @@ import (
 )
 
 const (
-	NamespaceID                       = "namespaceId"
-	ProjectID                         = "projectId"
-	ClusterID                         = "clusterId"
-	GlobalAdmin                       = "admin"
-	GlobalRestrictedAdmin             = "restricted-admin"
-	ClusterCRDsClusterRole            = "cluster-crd-clusterRole"
-	RestrictedAdminClusterRoleBinding = "restricted-admin-rb-cluster"
-	ProjectCRDsClusterRole            = "project-crd-clusterRole"
-	RestrictedAdminProjectRoleBinding = "restricted-admin-rb-project"
-	RestrictedAdminCRForClusters      = "restricted-admin-cr-clusters"
-	RestrictedAdminCRBForClusters     = "restricted-admin-crb-clusters"
+	NamespaceID            = "namespaceId"
+	ProjectID              = "projectId"
+	ClusterID              = "clusterId"
+	GlobalAdmin            = "admin"
+	ClusterCRDsClusterRole = "cluster-crd-clusterRole"
+	ProjectCRDsClusterRole = "project-crd-clusterRole"
```

Comment:
`ClusterCRDsClusterRole` and `ProjectCRDsClusterRole` are not used anymore. Can you remove them, please?

Reply:
Good catch, removed